src/js/components/tour/plan.tsx

```diff
@@ -10,14 +10,52 @@ import GuidedTour, {
 
 const PlanTour = (props: TourProps) => {
   const steps: Step[] = [
+    {
+      target: '.tour-project-tasks-list',
+      title: i18n.t('List of Tasks'),
+      content: (
+        <Trans i18nKey="walkthroughPlanListTasks">
+          Select the Tasks tab to see a list of all the work being done on this
+          Project and who is doing it. Tasks represent small changes to the
+          Project and may be grouped with other Tasks in an Epic.
+        </Trans>
+      ),
+      placement: 'right',
+      disableBeacon: true,
+    },
+    {
+      target: '.tour-create-task',
+      title: i18n.t('Create a Task to contribute'),
+      content: (
+        <Trans i18nKey="walkthroughPlanCreateTask">
+          To get started contributing to this Project, create a Task. Tasks
+          represent small changes to this Project; each one has a Developer and
+          a Tester. Tasks are equivalent to GitHub branches.
+        </Trans>
+      ),
+      placement: 'right',
+      disableBeacon: true,
+    },
+    {
+      target: '.tour-project-epics-list',
+      title: i18n.t('List of Epics'),
+      content: (
+        <Trans i18nKey="walkthroughPlanListEpics">
+          Select the Epics tab to see a list of all the Epics for this Project.
+          Each Epic is a group of related Tasks.
+        </Trans>
+      ),
+      placement: 'right',
+      disableBeacon: true,
+    },
```

Comment:
@SondraE @dvdherron While playing around with this today, I discovered some annoying limitations of react-joyride that are difficult to work around. Basically it skips a step if the highlighted element is not already visible, and there aren't good options for *first* making the element visible (e.g. switching the active tab) and *then* going to the next step. So, as my proposed workaround, I added steps for the "List of Tasks" and "List of Epic" *before* any time we need to switch the tab to highlight something in the other one. Practically this just meant adding two steps to the Plan walkthrough, and I think it's somewhat helpful to highlight the tabs before switching anyway. (I also had to remove the "Back" option.)

I hope this is acceptable. It's kinda hard to explain, so I'm happy to demo and discuss this on a call if that'd be useful.

Reply:
This seems like a good workaround for the react-joyride limitations. Was using the "Back" button related? The tour would skip highlighted elements that were no longer visible? I think it's fine to not have a "Back" button since there aren't so many steps.

Reply:
> This seems like a good workaround for the react-joyride limitations. Was using the "Back" button related? The tour would skip highlighted elements that were no longer visible? I think it's fine to not have a "Back" button since there aren't so many steps.

Yeah, removing the "back" button is related because we can only ensure an element is made visible *before* it is highlighted if we have a consistent order of steps. That is, "list of tasks" -> "create task" -> "list of epics" can work if we make "create task" visible when "list of tasks" is active. But reversing it would be more complicated, because we'd have to make "create task" visible when "list of epics" is active.

> And I'm fine with leaving out those additional Scratch Org steps if Sondra gives it the 👍🏽

I'm not proposing we leave them out entirely -- just that this PR stands on its own, and we can add them in a new PR.

Reply:
That makes sense. And I'm fine with adding those steps later @jgerigmeyer